NAS2D/Mixer/Mixer.h

```diff
@@ -19,18 +19,14 @@ namespace NAS2D
 	class Sound;
 	class Music;
 
-
 	/**
-	 * Mixer base class.
-	 *
-	 * Provides a standard Mixer interface. The base Mixer can be used
-	 * but will act as a NULL interface.
-	 */
+	Mixer base class.
+	*/
```

Comment:
I have some minor concerns about the style change here that removes the `*` from the left of comment blocks. This issue seems to involve Visual Studio tooling, so perhaps @ldicker83 would be in a better position to comment.

Reference to discussion thread:
https://github.com/lairworks/nas2d-core/pull/923#discussion_r670109187

I guess I'm not sure if we want to change style based on peculiar quirks of one particular tool. Or maybe we do. I'm a bit hesitant though if the original style was widespread.

----

Actually, now that it's a single line comment, using a single line comment block might sidestep the concern.
```cpp
/// Mixer base class.
```

Err, wait, that's kind of a useless comment, isn't it? Maybe it can just be removed entirely? That would also sidestep the problem.

Anyway, the discussion of style might still be relevant to future cases.